tests/functional/monitoring/prometheus/alerts/test_alert_mds_cache_high_usage.py

```diff
@@ -0,0 +1,234 @@
+import time
+import logging
+import pytest
+
+from ocs_ci.framework.testlib import tier2
+from ocs_ci.ocs import constants
+from concurrent.futures import ThreadPoolExecutor
+from ocs_ci.helpers import helpers
+from ocs_ci.ocs import cluster
+from ocs_ci.utility import prometheus
+from ocs_ci.framework.pytest_customization.marks import blue_squad
+from ocs_ci.utility.utils import ceph_health_check_base
+from ocs_ci.ocs.node import (
+    unschedule_nodes,
+    drain_nodes,
+    schedule_nodes,
+)
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope="function")
+def run_metadata_io_with_cephfs(pvc_factory, dc_pod_factory):
+    """
+    This function facilitates
+    1. Create PVC with Cephfs, access mode RWX
+    2. Create dc pod with Fedora image
+    3. Copy helper_scripts/meta_data_io.py to Fedora dc pod
+    4. Run meta_data_io.py on fedora pod
+    """
+    access_mode = constants.ACCESS_MODE_RWX
+    file = constants.METAIO
+    interface = constants.CEPHFILESYSTEM
+    log.info("Checking for Ceph Health OK")
+    ceph_health_check_base()
+
+    for i in range(3):
+        # Creating PVC with cephfs as interface
+        log.info(f"Creating {interface} based PVC")
+        pvc_obj = pvc_factory(interface=interface, access_mode=access_mode, size="30")
+        # Creating a Fedora dc pod
+        log.info("Creating fedora dc pod")
+        pod_obj = dc_pod_factory(
+            pvc=pvc_obj, access_mode=access_mode, interface=interface
+        )
+        # Copy meta_data_io.py to fedora pod
+        log.info("Copying meta_data_io.py to fedora pod ")
+        cmd = f"oc cp {file} {pvc_obj.namespace}/{pod_obj.name}:/"
+        helpers.run_cmd(cmd=cmd)
+        log.info("meta_data_io.py copied successfully ")
+
+        # Run meta_data_io.py on fedora pod
+        log.info("Running meta data IO on fedora pod ")
+        metaio_executor = ThreadPoolExecutor(max_workers=1)
+        metaio_executor.submit(
+            pod_obj.exec_sh_cmd_on_pod, command="python3 meta_data_io.py"
+        )
+
+
+def active_mds_alert_values(threading_lock):
+    active_mds = cluster.get_active_mds_info()["mds_daemon"]
+    sr_mds = cluster.get_mds_standby_replay_info()["mds_daemon"]
+    cache_alert = constants.ALERT_MDSCACHEUSAGEHIGH
```

Comment:
since you are calling it only once, you can directly call the constant in the function

Reply:
Anyhow I have to import constants for other variables so I just added it to the alerts section where everyone added the alerts in the constants.py. I can hard code it by just giving alert name within this function. I feel not to do that way because in future if someone want that alert to reuse they can refer alerts section in constant.py